### PR TITLE
Doc/update

### PR DESCRIPTION
**Pull request by chrpetre**
add PyTwinLogLevel documentation
Update ansys_dt.png
add download_file and load_data functions documentation to the published one

### CODE REVIEW

File: src/ansys/pytwin/examples/downloads.py [hunk 1]
```diff
@@ -146,8 +145,25 @@ def download_file(
...
 def load_data(inputs: str):
-    """Load a CSV input file into a Pandas Dataframe. Inputs is the path of the CSV file to be loaded,
-    containing the Time column and all the Twin inputs data"""
+    """
+    Load a CSV input file into a Pandas Dataframe.
+
+    Parameters
+    ----------
+    inputs : str
+        Path of the CSV file to be loaded, containing the Time column and all the Twin inputs data
```
Comment (MaxJPRey) [inline src/ansys/pytwin/examples/downloads.py:154]:
```suggestion
        Path of the CSV file to be loaded, containing the Time column and all the Twin inputs data.
```

File: src/ansys/pytwin/examples/downloads.py [hunk 2]
```diff
@@ -146,8 +145,25 @@ def download_file(
...
+
+    Returns
+    -------
+    inputs_df: pandas.DataFrame
+        A pandas.DataFrame storing time values as well as all the corresponding input data
```
Comment (MaxJPRey) [inline src/ansys/pytwin/examples/downloads.py:159]:
```suggestion
        A ``pandas.DataFrame`` storing time values as well as all the corresponding input data.
```

**Review by lboucin — APPROVED** [review]
Changes reviewed. LGTM

**Comment by lboucin** [comment]
@chrpetre we have an issue with the link added to the API doc. See below:
image
Doc creation log says:
image
I'll try to fix it and keep you posted.

**Comment by chrpetre** [comment]
@lboucin I have uploaded the missing files - rerunning all the workflows, thanks!

**Review by lboucin — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**